documentation/src/docs/asciidoc/release-notes/release-notes-5.6.0-M1.adoc

```diff
@@ -28,6 +28,8 @@ on GitHub.
 
 * New `printFailuresTo(PrintWriter, int)` method in `TestExecutionSummary` that allows one
   to specify the maximum number of lines to print for exception stack traces.
+* New `nullSymbols` attribute in `@CsvFileSource` and `@CsvSource`.
```

Comment:
This belongs in the JUnit Jupiter section, not the JUnit Platform section.

Reply:
I moved the documentation to the correct section.